src/components/PostListVolunteer/index.css

```diff
@@ -0,0 +1,58 @@
+.searchAndFilter {
+  border: 3px solid;
+  font-family: Acme;
+  color: #590995;
+  border-color: #590995;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  align-content: center;
+  padding: 1%;
+}
+
+.volunteerFilterLabel {
+  margin: 0.4em;
+}
+
+.resultsCountLabel {
+  color: #03c4a1;
+}
+
+.postSearch {
+  border: 0.1em solid #590995;
+  border-radius: 1em;
+  width: 20%;
+  background-image: url("https://img.icons8.com/fluent-systems-filled/24/000000/search.png");
+  background-position: right;
+  background-repeat: no-repeat;
+}
+
+.filter {
+  border: 0.1em solid #590995;
+  border-radius: 1em;
+}
+
+.displaySettings {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  align-content: center;
+  font-family: Acme;
+  color: #03c4a1;
+  margin: 1.5%;
+  margin-top: 2%;
+}
+
+.sort {
+  border: 0.1em solid #03c4a1;
+  border-radius: 1em;
+  color: #03c4a1;
+}
+
+.posts {
```

Comment:
classNames are not very clear, it can be changed to something more meaningful. this includes filter sort title and almost all one word css